llvm/include/llvm/Transforms/Instrumentation/PGOInstrumentation.h

```diff
@@ -53,15 +53,18 @@ class PGOInstrumentationGenCreateVar
   bool ProfileSampling;
 };
 
+enum class PGOInstrumentationType { Invalid = 0, FDO, CSFDO, CTXPROF };
```

Comment:
I think the enum members should have a prefix based on guideline here: https://llvm.org/docs/CodingStandards.html#the-low-level-issues

Also camel case (though FDO is an acronym so I'm not sure).

Reply:
The guideline is for cases where one can ambiguously reference the enum member directly, hence the "unless defined in their own small namespace or inside a class" where one's forced to prefix with the container name (esp. in the case of a class). That's why I made it an `enum class`.